src/api/middleware/validators/offer.js

```diff
@@ -53,7 +53,20 @@ const create = useExpressValidators([
 
     body("jobMaxDuration", ValidationReasons.DEFAULT)
         .optional()
-        .isInt().withMessage(ValidationReasons.INT),
+        .isInt().withMessage(ValidationReasons.INT)
+        .custom((jobMaxDuration, { req }) => {
+            try {
+                const { jobMinDuration } = req.body;
+
+                if (jobMinDuration >= jobMaxDuration) {
+                    throw new Error(ValidationReasons.MUST_BE_AFTER("jobMinDuration"));
+                }
+            } catch (err) {
+                console.error(err);
+                throw err;
+            }
+            return true;
+        }),
```

Comment:
```suggestion
const jobMaxDurationGreaterOrEqualThanJobMinDuration = (jobMaxDuration, { req }) => {
            try {
                const { jobMinDuration } = req.body;

                if (jobMinDuration >= jobMaxDuration) {
                    throw new Error(ValidationReasons.MUST_BE_AFTER("jobMinDuration"));
                }
            } catch (err) {
                console.error(err);
                throw err;
            }
            return true;
        }


...
        .custom(jobMaxDurationGreaterOrEqualThanJobMinDuration),
```

Move this to a function so that we can name the validation and make it clearer